

This phylogeny shows evolutionary relationships of SARS-CoV-2 viruses in Hawaii. Although the genetic relationships among sampled viruses are quite clear, there is considerable uncertainty surrounding estimates of specific transmission dates. 

Site numbering and genome structure uses [Wuhan-Hu-1/2019](https://www.ncbi.nlm.nih.gov/nuccore/MN908947) as reference. The phylogeny is rooted relative to early samples from Wuhan. Temporal resolution assumes a nucleotide substitution rate of 8 &times; 10^-4 subs per site per year. Mutational fitness is calculated using results from [Obermeyer et al](https://www.science.org/doi/10.1126/science.abm1208). Full details on bioinformatic processing can be found [here](https://github.com/nextstrain/ncov). 

Four different focus sampling levels by year (based on date of collection) are available via the "Dataset" dropdown on the left or by clicking on the following links: [2020-21](/ncov/hawaii/2020-21) | [2022a](/ncov/hawaii/2022a) | [2022b](/ncov/hawaii/2022b) | [2023](/ncov/hawaii/2023). To allow for overlap between the different focus sampling levels, each includes the last month of samples (based on date of collection) from the prior focus sampling level year or date range. 

Due to QC filtering parameters, not all sequenced samples from Hawaii for the respective time period are shown. 

Phylogenetic context of SARS-CoV-2 in SARS-related betacoronaviruses can be seen [here](https://nextstrain.org/groups/blab/sars-like-cov).

We gratefully acknowledge the authors, originating, and submitting laboratories of the genomic sequences and metadata made available through [GISAID](https://gisaid.org) on which this research is based. A full listing of all originating and submitting laboratories is available below. An attribution table is available by clicking on "DOWNLOAD DATA" at the bottom of the page and then clicking on "AUTHOR METADATA (TSV)" in the resulting dialog box.
